chrome-extension/manifest.js

```diff
@@ -64,7 +65,7 @@ const manifest = Object.assign(
       },
     ],
   },
-  !isFirefox && { side_panel: { ...sidePanelConfig.side_panel } },
+  !isFirefox && sidePanelConfig,
```

Comment:
```suggestion
  sidePanelConfig,
```
We already have logic for FireFox's sidePanel permission, so I think this is unnecessary.

Reply:
OO, in this case, it's completely unnecessary, but i remember there's some warnings in console, without that, i need to check it 1 more time, and i will adjust code 😄

Reply:
![image](https://github.com/user-attachments/assets/0a959e0f-ecb0-4057-abe6-3e4fee65a86e)

It's still doesn't supported, this check need to be here

Reply:
I removed that unnecessary check in config